hack/update-codegen.sh

```diff
@@ -4,4 +4,6 @@ chmod +x vendor/k8s.io/code-generator/generate-groups.sh
 ./vendor/k8s.io/code-generator/generate-groups.sh all \
       github.com/openshift/sriov-network-operator/pkg/client \
       github.com/openshift/sriov-network-operator/pkg/apis \
-      sriovnetwork:v1
+      sriovnetwork:v1 \
+      --go-header-file hack/boilerplate.go.txt
+chmod -x vendor/k8s.io/code-generator/generate-groups.sh
```

Comment:
we already have this line here https://github.com/openshift/sriov-network-operator/pull/270/files#diff-5345a009f642e55fc0d5b16d4f5d6f9bR3

Reply:
@SchSeba I opened this link, but it didn't switch to a certain line. could you point me to the duplicate code?

Reply:
For line 9, I remove the execution persmission from the generate-groups.sh file that was previously granted before executing the script. This avoid developer from wrongly submitting the file permission change.